pkg/gateway/model_build_rule_test.go

```diff
@@ -1453,6 +1464,8 @@ func Test_RuleModelBuild(t *testing.T) {
 			expectedSpec: []model.RuleSpec{
 				{
 					StackListenerId: "listener-id",
+					PathMatchPrefix: true,
+					PathMatchValue:  "/",
 					Action: model.RuleAction{
```

Comment:
Do we have any tests for empty GRPCRoute rule?

Reply:
We have a exact e2e test case that creates GRPCRouteRule with no match.